api/client/modelmanager/modelmanager.go

```diff
@@ -606,10 +608,9 @@ func (c *Client) ChangeModelCredential(model names.ModelTag, credential names.Cl
 // ValidateModelUpgrade checks to see if it's possible to upgrade a model,
 // before actually attempting to do the real model-upgrade.
 func (c *Client) ValidateModelUpgrade(model names.ModelTag, force bool) error {
-	if bestVer := c.BestAPIVersion(); bestVer < 9 {
-		return errors.NotImplementedf("ValidateModelUpgrade in version %v", bestVer)
+	if bestVer := c.BestAPIVersion(); bestVer != 9 {
```

Comment:
i don't quite understand the change to use `!=` instead of `<`
should it be `< 10` like below?

Reply:
Because `ValidateModelUpgrade` was added in 9 and is removed in 10 now.